### ISSUE TITLE
[BLOCKED][Housekeeping] Add auto merge capabilities to the repo

### ISSUE THREAD

**Issue by jakobe** [kirbydesign/designsystem#1366]
**Short description of housekeeping request**
Currently we manually have to merge approved PRs as well as manually updating each PR with latest master (whenever another PR gets merged to master).
This potentially also triggers redundant builds/deployments via GitHub Actions (as only 1 PR gets to be the next one merged to master and all other PRs again have to be updated with that change and thereby rebuild/-deployed).

**Alternatives**
Keep doing it manually and waste time :)

**Additional context**
- Create `merge:ready` and `automerge` labels
- Add new Github action  - see https://github.com/jakobe/github-actions-test/blob/master/.github/workflows/merge-queue.yml

### Kick Off

* [x] Ensure this issue has been `Tech refined` with @kirbydesign/kirby-guild and is updated with a clear implementation description
  _This issue should be in the [Ready to do](https://github.com/kirbydesign/designsystem/projects/1#column-4590936) column of the [Kirby kan-ban board](https://github.com/kirbydesign/designsystem/projects/1) before starting implementation_
* [x] Assign yourself to this issue and move it to the [In progress](https://github.com/kirbydesign/designsystem/projects/1#column-4590937) column of the [Kirby kan-ban board](https://github.com/kirbydesign/designsystem/projects/1)

### Code

* [x] Create Feature Branch from [master branch](https://github.com/kirbydesign/designsystem/tree/master)
* [ ] Implement/update unit tests
* [x] Create a draft implementation and push to Github
* [x] Ask a member of @kirbydesign/kirby-guild for a WIP review by creating a draft Pull Request

### Code Review

* [x] Open a pull request (or mark the existing draft PR as `Ready for review`) and ask @kirbydesign/kirby-guild for a review
  _Remember to add `closes #issueno` to the description of the PR._
* [ ] Once approved, merge feature branch/PR to master

:tada: Celebrate

**Comment by MadsBuchmann** [comment]
As GitHub has their [own auto merge functionality](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/automatically-merging-a-pull-request) it has been decided to use this instead and close the issue. 

If there is a need for functionality that does not exist in Githubs existing solution; a new issue should be created.
